src/main/java/frc/robot/subsystems/ConditionedDriveSubsystem.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.usfirst.frc.team4999.utils.Utils;
+
+public class ConditionedDriveSubsystem extends SubsystemBase {
```

Comment:
Should this be a Subsystem? Are we using any of the features of SubsystemBase?
Might be better to have it as a plain Java class.